main.py

```diff
@@ -83,13 +83,20 @@ def get_losses(task, mse_weight, pearson_weight, gpu):
     """
 
     if task == "regression":
-        loss_func = MultiLoss(['mse', 'pearsonloss'], [
-                              mse_weight, pearson_weight], device=gpu)
+        if poisson_weight > 0:
```

Comment:
Since there are several conditionals in this function now, wondering if the following re-write makes it more readable -

```
reg_loss_func = MultiLoss('poissonloss', poisson_weight, device=gpu) if poisson_weight > 0 else MultiLoss(['mse', 'pearsonloss'], [mse_weight, pearson_weight], device=gpu)
cla_loss_func = MultiLoss('bce', 1, device=gpu)

if task == "regression":
    loss_func = reg_loss_func
elif task == "classification":
    loss_func = "cla_loss_func
else:
    loss_func = [reg_loss_func, cla_loss_func]

return loss_func
```

Reply:
replaced